src/tools/diagnostic-messages.md

```diff
@@ -16225,6 +16312,40 @@ void f(int x) {
 }
 {% endprettify %}
 
+### positional_field_in_object_pattern
+
+_Object patterns can only use named fields._
+
+#### Description
+
+The analyzer produces this diagnostic when an object pattern contains a
+field that doesn't have a getter name. The fields provide a pattern to
+match against the value returned by a getter, and not specifying the name
+of the getter means that there's no way to access the value that the
+pattern is intended to match against.
```

Comment:
**issue:** Second sentence is too long (39 words) and lacks readability. (Grade 13)
```suggestion
The analyzer returns this message when an object pattern contains
a field without a getter name. The fields provide a pattern to match.
This pattern matches against the value that a getter returns.
 If you don't provide the name of the getter,
 you can't access the value that the pattern matches.
```

This change drops readability to Grade 7.